test/WebSocketsTestApp/runtimeconfig.template.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Yeah, I get the trade off. But I thought this was in public bits already....if not, my mistake.